tests/ui/duplicate_entry_error.rs

```diff
@@ -1,4 +1,4 @@
-// normalize-stderr-test "loaded from .*libstd-.*.rlib" -> "loaded from SYSROOT/libstd-*.rlib"
+// normalize-stderr-test "loaded from .*libstd-.*.rmeta" -> "loaded from SYSROOT/libstd-*.rmeta"
```

Comment:
The `.rmeta` file is now considered the canonical one as the `.rlib` doesn't contain any crate metadata beyond the header.